flink-table/flink-table-planner-blink/src/main/java/org/apache/flink/table/planner/operations/SqlToOperationConverter.java

```diff
@@ -159,6 +175,47 @@ private Operation convertDropTable(SqlDropTable sqlDropTable) {
 		return new DropTableOperation(identifier, sqlDropTable.getIfExists());
 	}
 
+	/** Convert CREATE FUNCTION statement. */
+	private Operation convertCreateFunction(SqlCreateFunction sqlCreateFunction) {
+		FunctionLanguage language = CatalogFunctionUtils.parseLanguage(sqlCreateFunction.getFunctionLanguage());
```

Comment:
just put `parseLanguange` in SqlToOperationConverter.java ? I feel flink and blink planner can decide whether they support python function independently